### DISCUSSION TITLE
Serverless Air Quality / Temp / Humidity monitor with realtime SPA dashboard

### DISCUSSION THREAD

**Discussion by barakplasma** [arendst/Tasmota#14722] [Show and tell]
Read more at https://dev.to/barakplasma/showdev-serverless-iot-dashboard-2h92

tl;dr
I used an ESP-8266 running Tasmota with a BME 680 air quality sensor to publish data to an MQTT broker. I also created a frontend dashboard which connects to the MQTT broker/topic and graphs/saves the data to the browser in realtime. (Serverless)

Code for the Realtime dashboard with graph and persistent data in indexeddb https://github.com/barakplasma/Browser-MQTT-Dashboard

**Comment by Derek-K** [comment]
It would be awesome if someone figured out a way to use the [BSEC library](https://www.bosch-sensortec.com/software-tools/software/bme680-software-bsec/) even if many drivers may need to be excluded...  ;)

**Reply by sfromis** [reply]
As an open source project, Tasmota cannot incorporate closed source libraries.
Still, you may be interested in https://github.com/arendst/Tasmota/discussions/23303
